_assets/js/laws-regs-parser.js

```diff
@@ -43,7 +43,9 @@ export default function parseLawsAndRegs (mainContent) {
 
 
     parent.insertBefore(newMainContent, mainContent);
-    mainContent.remove()
+    mainContent.remove();
+    const searchBtn = document.getElementById('searchbtn');
```

Comment:
suggestion: use class instead of ids to make this reusable (imagine we need a second search bar on the bottom of the page, for example)